sig-docs/blog-subproject/README.md

```diff
@@ -22,73 +22,4 @@ Regular Blog Meeting: Tuesdays at 18:30 UTC (biweekly). [Convert Your Timezone](
-
-To submit a blog post, follow the steps below.
-
-1. [Sign the CLA](https://kubernetes.io/docs/contribute/start/#sign-the-cla) if you have not yet done so.
-1. Have a look at the Markdown format for existing blog posts in the [website repository](https://github.com/kubernetes/website/tree/master/content/en/blog/_posts).
-1. Write out your blog post in a text editor of your choice.
-1. On the same link from step 2, click the Create new file button. Paste your content into the editor. Name the file to match the proposed title of the blog post, but don’t put the date in the file name. The blog reviewers will work with you on the final file name and the date the blog will be published.
-1. When you save the file, GitHub will walk you through the pull request process.
-1. A blog post reviewer will review your submission and work with you on feedback and final details. When the blog post is approved, the blog will be scheduled for publication.
-
-### Blog Guidelines
-
-Requested Content (with examples):
-
-- New Kubernetes capabilities
-- Kubernetes projects updates
-- Updates from Special Interest Groups
-- Tutorials and walkthroughs
-- Thought leadership around Kubernetes
-- Kubernetes Partner OSS integration
-- **Original content only**
-
-Unsuitable Content:
-
-- Vendor product pitches
-- Partner updates without an integration and customer story
-- Syndicated posts (language translations ok)
-
-## Review Process
-
-Once a blog post is submitted either via the form or a PR, it will be routed to the editorial team for review either via email for Google Docs or auto-assigning for a PR.
-
-Each blog post requires a LGTM from one copy editor, technical editor\*, and blog community manager. Once the necessary LGTMs are in place, an Editorial Lead will schedule and approve the blog post.
-
-_If a blog post does not contain any technical content (for example, [How You Can Help Localize Kubernetes Docs](https://kubernetes.io/blog/2019/04/26/how-you-can-help-localize-kubernetes-docs/)), the technical review can be omitted._
-
-### Embargoed Content
-
-The blog repository on GitHub is public, therefore any content that needs to remain confidential until a certain time (for example: release posts, security vulnerabilities) should be proposed by email message to [blog@kubernetes.io](mailto:blog@kubernetes.io). In your message, please note the time that the embargo will be lifted.
-
-### SLA
-
-Blog posts can take up to **2 weeks** to review. If you’d like to request an expedited review, please email [blog@kubernetes.io](mailto:blog@kubernetes.io).
-
-## Ongoing Blog Maintenance
-
-SIG-Docs approvers for English content can approve edits after the fact such as: broken links, copy edits, etc. However, approval and editorial review for new blog posts is limited to the Blog Team.
-
-We typically do not make edits to blog posts more than 1 years old.
```

Comment:
I would _log an issue_ to mention this in the docs contribution guide.
(We don't have good docs about maintaining existing content, in general).

Reply:
I created [Issue #31921](https://github.com/kubernetes/website/issues/31921). However, since I have [PR #31920](https://github.com/kubernetes/website/pull/31920) open to work on that page, do you think I could just add this content in there as well?

Reply:
Since [PR #31920](https://github.com/kubernetes/website/pull/31920) has now merged, I'll do a new PR to create a new "Updating existing content" page as you suggested [here](https://github.com/kubernetes/website/issues/31921#issuecomment-1057034524). Unless there is something else, then, I think this one can merge.